lms/djangoapps/course_structure_api/v0/api.py

```diff
@@ -0,0 +1,94 @@
+"""
+API implementation of the Course Structure API for Python code.
+
+Note: The course list and course detail functionality isn't currently supported here because of the tricky interactions between DRF and the code.
+Most of that information is available by accessing the course objects directly.
+"""
+
+from course_structure_api.v0 import serializers
+from course_structure_api.v0.errors import CourseNotFoundError, CourseStructureNotAvailableError
+from openedx.core.djangoapps.content.course_structures import models, tasks
+from courseware import courses
+
+
+def _retrieve_course(course_key):
```

Comment:
API changes look fine. Although, I am the de facto owner of the API, I have no plans to touch it in the near future. I'm happy to transfer ownership to someone else more active in the code and interested in getting this to a proper v1 status. I believe public documentation is the only thing lacking on that front.